src/libs/actions/Policy.js

```diff
@@ -245,7 +245,7 @@ function createOptimisticAnnounceRoomMembers(policyID, invitedEmailsToAccountIDs
 }
 
 /**
- * Build optimistic data for removing users from announce room
+ * Build optimistic data for removing users to the announce room
```

Comment:
For some reason in English we usually say "add to" and "remove from". I think of it like "to" means inserting and "from" means extracting.

So above we want "Build optimistic data for adding members to the announce room" and here we want "Build optimistic data for removing users from the announce room".

Sorry to be picky with but it's part of the checklist.